### PR TITLE
Symantec Endpoint Security C2C doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create release a new doc and release note for the Symantec Endpoint Security C2C.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[C2C - Symantec Endpoint Security (SES) - Update Docs](https://sumologic.atlassian.net/browse/CONN-3144)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import MyComponentSource from '!!raw-loader!/files/c2c/symantec-endpoint-security/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/symantec-endpoint-security/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/symantec-logo.svg')} alt="icon" width="125"/>
+
+Symantec Endpoint Protection is a client-server solution that protects laptops, desktops, and servers in network against malware, risks, and vulnerabilities. Symantec Endpoint Protection combines virus protection with advanced threat protection to proactively secure the client computers against known and unknown threats. This integration collects these information using the [Symantec Incidents API](https://apidocs.securitycloud.symantec.com/#/doc?id=edr_incidents) and ingests them into Sumo Logic.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:18]:
```suggestion
Symantec Endpoint Protection is a client-server solution that protects laptops, desktops, and servers in networks against malware, risks, and vulnerabilities. Symantec Endpoint Protection combines virus protection with advanced threat protection to proactively secure client computers against known and unknown threats. This integration collects this information using the [Symantec Incidents API](https://apidocs.securitycloud.symantec.com/#/doc?id=edr_incidents) and ingests it into Sumo Logic.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Vendor configuration
+
+The Symantec Endpoint Security source requires you to provide an authentication token. To obtain the token, follow the steps below.
+1. Sign in to the [Symantec Endpoint Security](https://login.broadcom.com/) platform.
+1. Go to **Integration** and select **Client Applications**.
+1. Click on **Add Client Application**.
+1. Enter any name for the application and press the **Add** button. The client application details screen appears.
+1. Select required privileges for the client application and click **Save**.
+1. Click on **More Options icon** and select **Client Secret**.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:38]:
```suggestion
1. Click the **More Options** icon and select **Client Secret**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+2. In **Client Secret**, enter the Client Secret you generated from the Symantec Endpoint Security platform.
+3. (Optional) In **Initial LookBack**, enter the first collection start time. Default is 1 day and maximum is 30 days.
+4. **Include Events**. Select this checkbox if you want to collect events from incidents.
+5. (Optional) The **Polling Interval** is set for 24 hours by default. You can adjust it based on your needs.
+6. (Optional) **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in Create a Processing Rule.
+7. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:71]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

**Review by jpipkin1 — APPROVED** [review]
Nice work, @JV0812! I have only a few suggestions.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+<img src={useBaseUrl('img/send-data/symantec-logo.svg')} alt="icon" width="125"/>
+
+Symantec Endpoint Protection is a client-server solution that protects laptops, desktops, and servers in networks against malware, risks, and vulnerabilities. Symantec Endpoint Protection combines virus protection with advanced threat protection to proactively secure client computers against known and unknown threats. This integration collects this information using the [Symantec Incidents API](https://apidocs.securitycloud.symantec.com/#/doc?id=edr_incidents) and ingests it into Sumo Logic.
+
+## Data collected
+
+| Polling Interval | Data Source | Description |
+| :--- | :--- |  :--- |
+| 5 min |  [Token](https://api.sep.securitycloud.symantec.com/v1/oauth2/tokens)| Used to generate a bearer token to subsequent API calls. |
```
Comment (paulina-kruczek-sumo) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:24]: The line about token line can be removed.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| 5 min |  [Incidents](https://api.sep.securitycloud.symantec.com/v1/incidents)| Used to retrieve incidents and incident events. |
+| 5 min |  [Incident Events](https://api.sep.securitycloud.symantec.com/v1/incidents/events)| Used to retrieve incident related events based on time range filter. |
+
+## Setup
+
+### Vendor configuration
+
+The Symantec Endpoint Security source requires you to provide an authentication token. To obtain the token, follow the steps below.
```
Comment (paulina-kruczek-sumo) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:32]:
```suggestion
The Symantec Endpoint Security source requires you to provide credentials. To obtain the client secret, follow the steps below.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+1. Select required privileges for the client application and click **Save**.
+1. Click the **More Options** icon and select **Client Secret**.
+1. Copy the **OAuth Credentials** value.
+1. Run the below curl command with the required headers to generate the bearer token:
+    ```
+    curl -X POST https://api.sep.securitycloud.symantec.com/v1/oauth2/tokens
+    -H “accept: application/json”
+    -H “authorization: {{OAuth Credentials value}}"
+    -H “content-type: application/x-www-form-urlencoded”
+    ```
```
Comment (paulina-kruczek-sumo) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/symantec-endpoint-security-source.md:46]:
```suggestion

```

**Comment by JV0812** [comment]
Releasing this C2C ass Beta version.

**Pull request merged**